esmvaltool/diag_scripts/miles/miles_block.R

```diff
@@ -0,0 +1,133 @@
+# #############################################################################
+# miles_block.r
+# Authors:       P. Davini (ISAC-CNR, Italy) (author of MiLES)
+# 	         J. von Hardenberg (ISAC-CNR, Italy) (ESMValTool adaptation)
+#                E. Arnone (ISAC-CNR, Italy) (ESMValTool v2.0 adaptation)
+# #############################################################################
+# Description
+# MiLES is a tool for estimating properties of mid-latitude climate originally
+# thought for EC-Earth output and then extended to any model data.
+# It works on daily 500hPa geopotential height data and it produces
+# climatological figures for the chosen time period. Data are interpolated
+# on a common 2.5x2.5 grid.
+# Model data are compared against ECMWF ERA-INTERIM reanalysis
+# for a standard period (1989-2010).
+# It supports analysis for the 4 standard seasons.#
+# Required
+#
+# Optional
+#
+# Caveats
+#
+# Modification history
+#   20180525-arno_en: Conversion to v2.0
+#   20181203 hard_jo: Completed conversion, rlint compliant
+#
+# ############################################################################
+
+library(tools)
+library(yaml)
+
+spath <- "./esmvaltool/diag_scripts/miles/"
```

Comment:
Please make use of the `diag_scripts` environmental variable to find the location of the diagnostic scripts.

Reply:
Fixed in https://github.com/ESMValGroup/ESMValTool/pull/735/commits/96002b2ee5e04896056ee2cf517c8d8e82d7de23